packages/manager/modules/pci/src/projects/project/data-processing/translations/Messages_fr_FR.json

```
@@ -0,0 +1,117 @@
+  "data_processing_onboarding_guides_quick_start_title": "Lancer mon premier job Spark",
+  "data_processing_onboarding_guides_monitor_job_title": "Monitorer mes jobs",
+  "data_processing_submit_job_stepper_spark_driver_template_label": "Modèle de driver",
+  "data_processing_submit_job_stepper_spark_worker_template_label": "Modèle d'executor",
+  "data_processing_submit_job_stepper_spark_worker_count_label": "Nombre d'executors",
+  "data_processing_submit_job_stepper_spark_worker_count_description": "Sélectionnez le nombre d'executors pour votre job",
+  "data_processing_submit_job_stepper_spark_advanced": "Configuration avancée",
+  "data_processing_submit_job_stepper_spark_standard": "Configuration standard",
+  "data_processing_submit_job_stepper_spark_driver_cores_label": "Nombre de vCores du driver",
+  "data_processing_submit_job_stepper_spark_driver_cores_description": "",
+  "data_processing_submit_job_stepper_spark_driver_memory_label": "Mémoire du driver",
+  "data_processing_submit_job_stepper_spark_driver_memory_description": "",
+  "data_processing_submit_job_stepper_spark_driver_memory_overhead_label": "Memory overhead driver",
+  "data_processing_submit_job_stepper_spark_driver_memory_overhead_description": "",
+  "data_processing_submit_job_stepper_spark_worker_cores_label": "Nombre de vCores de l'executor",
+  "data_processing_submit_job_stepper_spark_worker_cores_description": "",
+  "data_processing_submit_job_stepper_spark_worker_memory_label": "Mémoire executor",
+  "data_processing_submit_job_stepper_spark_worker_memory_description": "",
+  "data_processing_submit_job_stepper_spark_worker_memory_overhead_label": "Memory overhead executor",
+  "data_processing_submit_job_stepper_spark_worker_memory_overhead_description": "",
+  "data_processing_submit_job_stepper_spark_estimated_price": "Prix estimé: ",
+  "data_processing_submit_job_stepper_spark_main_class_label": "Classe principale",
+  "data_processing_submit_job_stepper_spark_main_class_help": "ex. org.apache.spark.examples.SparkPi",
+  "data_processing_submit_job_stepper_spark_jar_file_label": "Fichier JAR",
+  "data_processing_submit_job_stepper_spark_python_file_label": "Fichier Pyhon",
+  "data_processing_submit_job_stepper_spark_jar_file_help": "Chemin vers votre fichier JAR au sein de votre conteneur d'objets",
+  "data_processing_submit_job_stepper_spark_job_name_label": "Nom du job",
+  "data_processing_submit_job_stepper_spark_job_name_help": "Minimum 3 caractères",
+  "data_processing_submit_job_stepper_spark_swift_label": "Conteneur Swift",
+  "data_processing_submit_job_stepper_spark_swift_placeholder": "Sélectionnez un conteneur",
+  "data_processing_submit_job_stepper_spark_version_label": "Version de Spark",
+  "data_processing_submit_job_stepper_spark_arguments_label": "Arguments (optionnel)",
+  "data_processing_submit_job_stepper_spark_arguments_placeholder": "Pressez la touche <Entrer> pour ajouter un argument",
```

```suggestion
  "data_processing_submit_job_stepper_spark_arguments_placeholder": "Pressez la touche <kbd>Entrer</kbd> pour ajouter un argument",
```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd

---

That translation is inside an input placeholder, I can't have tags inside it.